react-live.d.ts

```diff
@@ -0,0 +1,30 @@
+export as namespace ReactLive;
+
+import { Component } from 'react'
+
+export interface LiveProviderProps {
+  code: string;
+  error?: string;
+  onError?: Function;
+  onChange?: Function;
+  element?: any;
+}
+
+export class LiveProvider extends Component<LiveProviderProps, {}>{}
+
+export interface EditorProps extends HTMLElement {
```

Comment:
LiveEditor also accepts the props of Editor, so I don't know if `HTMLElement` includes `contentEditable` ^^